code/go/0chain.net/smartcontract/minersc/models.go

```diff
@@ -1218,13 +1221,34 @@ func (dmn *DKGMinerNodes) GetHashBytes() []byte {
 
 // getMinersList returns miners list
 func getMinersList(state cstate.StateContextI) (*MinerNodes, error) {
+
```

Comment:
The only changes to the smart contract code should be to emit events. 

Smartcontracts should have no access to the eventDb object, this is the reason for https://github.com/0chain/0chain/issues/657, to prevent people trying to use methods like GetEventDb in smart contracts.

Reply:
noted. So in this case is someone does GetEventDb() (even through not should be done), it should return nil, right?
That's the expected behavior?